apps/arweave/src/ar.erl

```diff
@@ -541,6 +547,14 @@ parse_cli_args(["cm_exit_peer", Peer | Rest], C) ->
 			io:format("Peer ~p is invalid.~n", [Peer]),
 			parse_cli_args(Rest, C)
 	end;
+parse_cli_args(["pool_server" | Rest], C) ->
+	parse_cli_args(Rest, C#config{ pool_server = true });
+parse_cli_args(["pool_client" | Rest], C) ->
+	parse_cli_args(Rest, C#config{ pool_client = true });
+parse_cli_args(["pool_api_key", Key | Rest], C) ->
+	parse_cli_args(Rest, C#config{ pool_api_key = list_to_binary(Key) });
+parse_cli_args(["pool_host", Host | Rest], C) ->
```

Comment:
Earlier I don't think we had both pool_server and pool_host, so renaming pool_proxy to pool_host IMO made sense. Now, though, since server and host are often used interchangeably in the industry the two variables are confusing. Is pool_server the node arweave node backing the pool_host? If so what about "pool_arweave_node" or something? Or rename pool_host to pool_gateway?

Reply:
Oh wait reading the ar.erl description what about:
is_pool_client (true/false)
is_pool_server (true/false)
pool_server_address (IP address)?

Reply:
Renamed